java/ingest/ingest-status-store/src/test/java/sleeper/ingest/status/store/job/QueryIngestJobStatusByPeriodIT.java

```diff
@@ -54,14 +54,14 @@ public void shouldReturnIngestJobsInPeriod() {
     public void shouldExcludeIngestJobOutsidePeriod() {
         // Given
         IngestJob job = jobWithFiles("file");
-        Instant startedTime = Instant.now();
+        Instant startedTime = Instant.parse("2023-01-03T14:50:00.001Z");
```

Comment:
These Instant.parse changes aren't quite valid because the update time is set in the status store and we're not controlling for that.

Reply:
Actually the update times aren't used this way. It's a specific problem for the failing test. I'll make a new comment.